vitest.config.ts

```diff
@@ -23,12 +23,46 @@ export default defineConfig((configEnv) =>
           provider: 'v8',
           reporter: ['lcov', 'html'],
           exclude: [
+            '**/*.d.ts',
+            '**/*.json',
+            '**/*.xml',
+            '**/*.yaml',
+            '**/*.md',
+            '**/*.html',
+            '**/*.css',
+            '**/*.properties',
             '*.config.*js',
             'node_modules/',
+            'browser-tests/',
+            'build/',
+            'codegen.ts',
             'src/index.tsx',
             'src/domain/api/generatedTypes',
             'public/mockServiceWorker.js',
             'src/setupTests.ts',
+            '**/__tests__/**',
+            '**/__snapshots__/**',
+            '**/*.test.ts',
+            '**/*.spec.ts',
+            'src/domain/eventGroups/api/eventGroupsApi.ts',
+            'src/domain/messages/api/messagesApi.ts',
+            'src/domain/ticketSystemPassword/api/ticketSystemPasswordsApi.ts',
+            'src/domain/occurrences/api/OccurrenceApi.ts',
+            'src/domain/eventsAndEventGroups/api/eventsAndEventGroupsApi.ts',
+            'src/domain/occurrences/api/OccurrenceApi.ts',
+            'src/domain/children/api/ChildApi.ts',
+            'src/domain/venues/api/VenueApi.ts',
+            'src/domain/events/api/EventApi.ts',
+            'src/domain/profile/api.ts',
+            'src/domain/dashboard/api.ts',
+            '**/*Query.ts',
+            '**/*Query.tsx',
+            '**/*Queries.ts',
+            '**/*Mutation.ts',
+            '**/*Mutation.tsx',
+            '**/*Mutations.ts',
+            '**/mutations/**',
+            '**/queries/**',
```

Comment:
It would be valuable IMHO to add a comment(s) here about why these files are excluded from code coverage.